storage/aptosdb/src/state_store/mod.rs

```diff
@@ -156,9 +157,9 @@ impl StateStore {
         key_prefix: &StateKeyPrefix,
         desired_version: Version,
     ) -> Result<HashMap<StateKey, StateValue>> {
-        let mut iter = self
-            .ledger_db
-            .iter::<StateValueSchema>(Default::default())?;
+        let mut read_opts = ReadOptions::default();
+        read_opts.set_total_order_seek(true);
```

Comment:
This is tricky.. please comment why this is used here and why it should not be set in the other place.

Reply:
hmm. okay-ish

Reply:
🙌